eap-session-replication/eap-configuration/standalone-ha.xml

```diff
@@ -0,0 +1,607 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<server xmlns="urn:jboss:domain:8.0">
```

Comment:
Is this configuration used? I mean whether it replaces `standalone-azure-ha.xml` If not it can be removed.

Reply:
This configuration is not used and hence I have deleted it